docs/user-guide/how-to.md

```diff
@@ -956,3 +956,10 @@ Interesting parameters include:
   load on your metrics database (through increased metric cardinality) and also
   by increasing how carefully you need to guard your metrics against
   unauthorized viewers.
+* `--exclude-objects-with-label`: present by default with label
```

Comment:
I'd move the note about the Rancher stuff to a subbullet at the end. I'd start by saying it has the same effect as [disabling propagation](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/how-to.md#limit-the-propagation-of-an-object-to-descendant-namespaces) but is useful when you can't control the labels on the objects. And then just mention at the end that the default deployment of HNC excludes Rancher.